bootstrap/bootstrap.php

```diff
@@ -139,25 +144,26 @@
     }
 }
 
-if (!defined('CRAFT_EPHEMERAL') || CRAFT_EPHEMERAL === false) {
-    $ensureFolderIsReadable($storagePath, true);
```

Comment:
It doesn't really make sense for `CRAFT_EPHEMERAL` to be a condition here…
Even in an ephemeral env, craft needs to be able to write to the storage dir.